app/graphql/types/record_type.rb

```diff
@@ -146,48 +146,55 @@ def matched_phrases
+      field :source, String, null: false,
+                             description: 'Name of source record system. One of the following:\n* MIT Alma\n* MIT ArchivesSpace\n* DSpace@MIT\n* Abdul Latif Jameel Poverty Action Lab Dataverse\n* Woods Hole Open Access Server\n* Zenodo'
+      field :source_link, String, null: false, description: 'URL for source record in source system'
+      field :title, String, null: false, description: 'Title of item'
+      field :alternate_titles, [Types::AlternateTitleType], null: true, description: 'Alternate titles for the item'
+      field :contributors, [Types::ContributorType], null: true, description: 'Contributors to the item'
+      field :subjects, [Types::SubjectType], null: true, description: 'Subject terms for item'
+      field :identifiers, [Types::IdentiferType], null: true,
+                                                  description: 'Unique identifiers associated with the item; e.g., ISBN, DOI, etc.'
       field :isbns, [String], null: true, deprecation_reason: 'Use `identifiers`'
       field :issns, [String], null: true, deprecation_reason: 'Use `identifiers`'
       field :dois, [String], null: true, deprecation_reason: 'Use `identifiers`'
       field :oclcs, [String], null: true, deprecation_reason: 'Use `identifiers`'
       field :lccn, String, null: true, deprecation_reason: 'Use `identifiers`'
       field :place_of_publication, String, null: true, deprecation_reason: 'Use `locations`'
-      field :languages, [String], null: true
+      field :languages, [String], null: true, description: 'Language(s) of item'
       field :publication_date, String, null: true, deprecation_reason: 'Use `dates`'
-      field :content_type, [String], null: true
-      field :call_numbers, [String], null: true
-      field :citation, String, null: true
-      field :edition, String, null: true
+      field :content_type, [String], null: true,
+                                     description: 'Type of content of item; e.g., "still image", "text", etc.'
+      field :call_numbers, [String], null: true, description: 'Identification number used to classify and locate item'
+      field :citation, String, null: true, description: 'Citation for item'
+      field :edition, String, null: true, description: 'Edition information for item'
       field :imprint, [String], null: true, deprecation_reason: 'Use `publicationInformation`'
-      field :publication_information, [String]
-      field :physical_description, String, null: true
-      field :publication_frequency, [String], null: true
+      field :publication_information, [String], description: 'Imprint information for item'
+      field :physical_description, String, null: true, description: 'Physical description of item'
+      field :publication_frequency, [String], null: true,
+                                              description: 'Publication frequency of item (used for serials)'
       field :numbering, String, null: true
```

Comment:
@hakbailey do you have suggestion as to what wording we might use to describe `numbering`? It wasn't defined in the data model documents used to add descriptions to the other fields.